### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Natemo6348** [infinitered/nsfwjs#499]
So let's say I wanted to use this with a discord bot that checks a url for nsfw content. Can I do that with this, and how could I do it?

**Comment by GantMan** [comment]
You'd have to have your bot pull the HTML/CSS and parse for JPG/PNG/GIF files.   Once you parse the files you would loop through them and use that information to report.

**Comment by GantMan** [comment]
You're not the first person to ask about this, perhaps I should make this a feature of the library?  Just thinking on this.  Perhaps it's a useful enough skill.

**Comment by Natemo6348** [comment]
Yeah, I have a command that screenshots whatever site you want, and if it's NSFW it returns `This site is NSFW`, but it doesn't check for images, just URLs

**Comment by GantMan** [comment]
Currently, to use this you'd have to parse the HTML/CSS.  I'll mark this ticket as a feature request and revisit it next Hacktoberfest.

**Comment by EvEloBE** [comment]
I've just added this to my discord bot as well. This is how you can do it yourself.
```
const nsfwjs = require("nsfwjs")
const fetch = require("node-fetch")

// Attachment has to contain image url, height and width
async function classifyImage(attachment) {
const model = await nsfwjs.load()

const imageBuffer = await fetch(attachment.url).then((res) => res.buffer());

return await model.classify({ data: imageBuffer, height: attachment.height, width: attachment.width })
}
```
This is a bit different from my implementation, but i think it should work.
